docs/etcd/cluster_setup.md

```diff
@@ -0,0 +1,253 @@
+
+## Start the cluster
+
+This section will guide you through installing, setting up and start a demo etcd cluster.
+
+Please note that while it will create a ready-to-use and working cluster, we **strongly** encourage you to read etcd's [official documentation](https://etcd.io/docs/v3.4.0/) to learn how to make it more robust, resilient and secure in case you want to use it in production.
+
+### Requirements
+
+Please make sure the machine where you plan to install etcd is reachable by the CN-WAN Operator, so that you can later follow the other guides on this documentation.
+
+Also, although this won't be much of problem, make sure that ports `2379` and `2380` are free on that machine, as those are well-known ports [assigned by IANA](https://www.iana.org/assignments/service-names-port-numbers/service-names-port-numbers.xhtml?search=etcd) to etcd and the ones that will be used throughout these guides. Nonetheless, you can also specify the port on installation phase.
+
+Please note that as of this writing, `v3.4.14` is the latest version of etcd and it has no installation/binaries for *Windows* and therefore we will be using *Linux*/*Mac* shell commands from now on. If you plan to use *Windows* you will need to either use [Docker](https://docs.docker.com/docker-for-windows/install/), [wsl](https://docs.microsoft.com/en-us/windows/wsl/install-win10) or use a virtual machine.
+
+### Get the IP address
+
+First, get the IP address of the machine. Depending on your operating system, this could mean running either `ip a` or `ifconfig -a` or even go through the machine's operating system settings if it has GUI.
+
+Choose an IP address that is reachable by your Kubernetes Cluster: you may even use *localhost* as the address but it won't be reachable by the CN-WAN Operator.
+
+Export the following variables, containing the IP address and the name of this demo:
+
+```bash
+export DEMO_IP=<your-IP-address>
+export DEMO_NAME=<name>
+```
+
+For example, supposing the address you chose is `10.10.10.10` and the name is `demo`, the command above would be:
+
+```bash
+export DEMO_IP=10.10.10.10
+export DEMO_NAME=demo
+```
+
+### Download an official release
+
+Visit etcd's [releases page](https://github.com/etcd-io/etcd/releases) and choose the installing method most appropriate for your operating system. For the scope of this demo, we recommend you installing etcd *without* Docker, as the commands will be shorter and easier to type and it will be the one used on this guide.
+
+Don't start etcd just yet, as we will do this in a bit. Please stop it if you already started it.
+
+Next, if you installed etcd locally follow the [next section](#start-etcd-without-docker), otherwise go to [Start etcd *with* docker](#start-etcd-with-docker).
+
+### Start etcd *without* docker
+
+In case you are running etcd *without* Docker and installed etcd from the releases page, start etcd as:
+
+```bash
+/tmp/etcd-download-test/etcd \
```

Comment:
Using `$ETCD_PATH` would be annoying to write imo. I will write an alias like I did on the other guide.

I'm also further reiterating that this is only a demo cluster and not intended to be used in production and encourage users to follow other better guides in case they want to use this in production.

Reply:
Sounds good then, I find this better now that the file is called "demo_cluster_setup"